src/elements/Banner/Banner.tsx

```diff
@@ -23,6 +23,10 @@ const Target = styled.div`
   }
 `
 
+const Wrapper = styled(Box)`
+  transition: background-color 250ms linear;
```

Comment:
Do we want to transition the background color? I might miss the conversations, but this feels a new UI pattern worth checking in with design.